src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/StartedNode.kt

```diff
@@ -68,9 +70,20 @@ class StartedNode(
             resultsTransport = this.resultsTransport,
             unpackedProduct = this.unpackedProduct,
             monitoringProcesses = this.monitoringProcesses,
-            ssh = this.ssh
+            ssh = this.ssh,
+            jdk = this.jdk
         )
     }
 
+    fun stopNode() {
```

Comment:
I though: Should it return `StoppedNode`? That would kinda introduce cyclic dependency, however would follow the architecture of returning up to date client related to the state after operation.

In the end I don't see it as a big deal, however I thought it's worth to highlight this puzzle.

Reply:
That would be nice to have. But the `StoppedNode` would need to be passed via the constructor. Every time we expose a constructor it's a huge pain. We probably need builders every time. AWS SDK does exactly that.

Reply:
Or maybe Java 9 module system could hide that stuff.